### ISSUE TITLE
Slider component won't work on touch+AT devices

### ISSUE THREAD

**Issue by patrickhlauke** [material-components/material-web#1285]
**Describe the bug**
While the slider seems to follow the ARIA 1.1 pattern correctly, it - as the pattern itself - doesn't work for users on a touch device using AT (e.g. iOS/VoiceOver, Android/TalkBack). The ARIA makes it announce like a native `<input type="range">`, and gives the hint to users that they can use gestures like "swipe up or down to change the value" (on iOS) / "use the volume buttons to change the value" (Android). However, as the slider can't actually intercept those (just yet anyway, until it uses Accessibility Object Model user action events https://wicg.github.io/aom/spec/#user-action-events), it is essentially broken for these users.

**To Reproduce**
1. On a mobile/tablet device, with AT running, navigate to the Slider example
2. Set focus to the slider
3. Attempt to change the value by following the prompt given by the AT (e.g. "swipe up or down" / "use volume keys" depending on the specific OS/AT)

**Expected behavior**
The slider values change

**Browser Version (please complete the following information):**
 - OS: iOS, Android
 - Browser: All

**Additional context**
The most robust way to make this universally work, currently, is to actually have a proper `<input type="range">` which is either heavily styled, or which is being "piggy-backed" via CSS/JS. See https://codepen.io/patrickhlauke/pen/byWPMX for demo and explanation

**Comment by e111077** [comment]
Duplicate #1279

**Comment by patrickhlauke** [comment]
not quite sure why this was marked as a duplicate, when this is specifically about touch + assistive tech, while #1279 is about mouse handling?

**Comment by e111077** [comment]
Ah oops I seem to have responded to the wrong browser tab!

**Comment by oliviertassinari** [comment]
@patrickhlauke Should we open an issue on https://github.com/w3c/aria-practices to report the issue with https://www.w3.org/TR/wai-aria-practices/examples/slider/slider-1.html?

**Comment by patrickhlauke** [comment]
already known issue with aria-practices ... sadly, not much movement has been made in that respect https://github.com/w3c/aria-practices/issues/8

**Comment by asyncLiz** [comment]
Closing for now since there isn't something actionable we can do about this.

I'd recommend trying the M3 slider, which may provide some better integration since it uses native sliders under the hood.
